app/models/user.rb

```diff
@@ -57,4 +54,30 @@ def daac_curator?
   def arc?
     admin || curator
   end
+
+  def self.from_omniauth(auth)
+
+    user = User.where(:provider => auth.provider, :uid => auth.uid).first
+    if !user
+      user = User.new
+      user.uid = auth.uid
+      user.role = 'arc_curator'
```

Comment:
is this an "enum" somewhere? what are the full possibilities of values?

Reply:
they are not enum but I like the idea of a technical ticket moving in this direction though.   The full possibilities are arc_curator, daac_curator, and admin

Reply:
Interesting, the closest it looks ruby has to enum is symbols...  But the value side of symbols can't be stored in a DB, I don't believe, so we'd have to change this implementation to use constants.  https://stackoverflow.com/questions/75759/enums-in-ruby

Reply:
What I'd do is include in the other refactoring ticket for roles, the use of constants, so this doesn't get lost.